src/sinks/mqtt/integration_tests.rs

```diff
@@ -63,10 +63,8 @@ async fn mqtt_happy() {
     while failures < 5 && out.len() < input.len() {
         if let Ok(try_msg) = tokio::time::timeout(Duration::from_secs(1), eventloop.poll()).await {
             let msg = try_msg.unwrap();
-            println!("Received = {:?}", msg);
-            out.push("test_str".to_string());
-            //let s = String::from_utf8_lossy(msg.data.as_slice()).into_owned();
-            //out.push(s);
+            let s = String::from_utf8_lossy(msg.data.as_slice()).into_owned();
```

Comment:
compilation error here

Reply:
The compilation error is fixed but the test still does not work since some env variable `MQTT_VERSION` is not defined. And I do not know, where it should be specified.

Reply:
That should be `CONFIG_VERSION`. Those settings changed in #16311. I'm not surprised that will have broken some outstanding PRs.